tests/end2end/features/spell.feature

```diff
@@ -0,0 +1,38 @@
+# vim: ft=cucumber fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
These tests look quite hacky (I'd like to avoid `:debug-pyeval` in feature tests), and I also don't see a reason why they need to be end2end tests (they don't really test for proper integration with QtWebEngine/Chromium, but they can't).

I think it'd be better if those were unit tests for `webenginesettings.py` instead. Then it's much easier to check the profile, and they also are much faster and more reliable that way.